sklearn/linear_model/stochastic_gradient.py

```diff
@@ -724,6 +724,14 @@ class SGDClassifier(BaseSGDClassifier):
         When set to True, reuse the solution of the previous call to fit as
         initialization, otherwise, just erase the previous solution.
 
+        Repeatedly calling fit or partial_fit when warm_start is True can
+        result in a different solution than when calling fit a single time
+        because of the way the data is shuffled.
+        If a dynamic learning rate is used, the learning rate is adapted
```

Comment:
Maybe adding a blank line above it? Otherwise it will not be perceived as a new paragraph. See https://14983-843222-gh.circle-artifacts.com/0/home/ubuntu/scikit-learn/doc/_build/html/stable/modules/generated/sklearn.linear_model.SGDClassifier.html